### PR TITLE
[Fix] Account removed from "Settings" is not removed from the drawer

### PR DESCRIPTION
**Pull request by fesave**
## Related Issues
App: #3340 

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: changelog/unreleased/3381 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Bugfix: Account removed is not removed from the drawer
+
+When an account was deleted from the phone settings, in the accounts section, it was not deleted from the Navigation Drawer,
+now when deleting an account from there, the Navigation Drawer refreshes and it is deleted correctly.
```
Comment (JuancaG05) [inline changelog/unreleased/3381:4]: Just a tiny change I suggest, but up to you 😉 
P.S.: I like the passive voice you used 👍 
```suggestion
When an account was deleted from the app settings, in the accounts section, it was not removed from the Navigation Drawer.
Now, when deleting an account from there, the Navigation Drawer is refreshed and the removed account is no more shown.
```

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Just a suggestion, but it's up to you 👍

File: changelog/unreleased/3381 (hunk 1 above)
Comment (fesave) [reply changelog/unreleased/3381:4]: Oops, you're wrong my friend 😄 , the problem appears when the user deletes his account from the phone settings (not from the app settings)

Thanks for the other suggestions, no problem to change them.

Comment (JuancaG05) [reply changelog/unreleased/3381]: 😱  My fault!! Sorry, thanks for the information

**Review by JuancaG05 — APPROVED** [review]

**Comment by jesmrec** [comment]
Fixed.

**Pull request merged**